app/poi_api/poi_rest_handler.py

```diff
@@ -0,0 +1,33 @@
+from .poi_service import POI_Service
+from flask import jsonify
+from .poi_errors import POINotFoundError, InvalidPOISuggestionError
+
+poi_service_class = POI_Service()
+
+# TODO Exception handling
+# TODO type hints
+def get_all_POI():
+    try:
+        list_all_poi = poi_service_class.get_all_POI()
+        return [poi.to_dict() for poi in list_all_poi], 200
+    except Exception as e:
+        return jsonify({"success": False, "message": str(e), "error_code": 400}), 400
+
+
+def get_POI(poi_id):
+    try:
+        get_poi = poi_service_class.get_POI(poi_id)
+        return get_poi.to_dict(), 200
+    except POINotFoundError as e:
+        return jsonify({"success": False, "message": str(e), "error_code": 404}), 404

```

Comment:
_If_ you have to jsonify the error it would be easiest to include a jsonify method on the error object itself so you can just call `e.jsonify()` instead of this construction each time.